unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -105,6 +105,21 @@ async UniTaskVoid LoadEmotesAsync(CancellationToken ct = default)
                     allEmotes.AddRange(embeddedEmotesSo.GetAllEmotes());
                     allEmotes.AddRange(await emotesCatalogService.RequestOwnedEmotesAsync(userProfileBridge.GetOwn().userId, ct) ?? Array.Empty<WearableItem>());
 
+                    Dictionary<string, WearableItem> consolidatedEmotes = new Dictionary<string, WearableItem>();
+                    foreach (var emote in allEmotes)
+                    {
+                        if (consolidatedEmotes.ContainsKey(emote.id))
+                        {
+                            consolidatedEmotes[emote.id].amount += emote.amount + 1;
```

Comment:
Why `+1` ? Shouldnt all owned nfts have an amount of `>= 1`?

Reply:
So, this snippet calculates how many emotes of the same type (same shortened urn) an user holds, that's why we iterate over allEmotes retrieved and consolidate the correct amount taking into account the `emote.id` (shortened urn) for each emote.

Reply:
We need to calculate the amount manually here.